src/entity/relation.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * # Relation Entity
+ *
+ * | property    | Type       | Default     |
+ * |-------------|------------|-------------|
+ * | _id         | ObjectID   |
+ * | fromId      | ObjectID   |
+ * | toId        | ObjectID   |
```

Comment:
perhaps you can simply set `_id` as `${fromId}_${toId}`, then catch conflict error and throw bad request.

Reply:
That will make geo query more difficult. And `fromId`/`toId` is ObjectId (for better geo relation query), which is fixed in 24 chars, `${fromId}_${toId}` is beyond 24 chars that could not become a `_id` of ObjectId.

So, I think it will be a good idea if we use that form with Redis cache before MongoDB queries.